.github/actions/publish-if-newer/action.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Publish if newer
+description: Publishes the package to crates.io if the version in the repo is different than the one on crates.io
+
+inputs:
+  package:
+    required: true
+    description: The name of the package to publish
+
+runs:
+  using: composite
+  steps:
+    - name: Publish
+      shell: bash
+      run: |
+        if [ $(curl https://crates.io/api/v1/crates/ink-wrapper | jq '.crate.newest_version') \
+          != $(cargo metadata | jq '.packages[] | select(.name == "ink-wrapper") | .version') ]
```

Comment:
So we would publish `ink-wrapper-types` only if `ink-wrapper`'s version is higher than in crates? i.e. in the action we compare `ink-wrapper`'s version from the crates. Why not compare `{{ inputs.package }}`?